app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
```

Comment:
- [ ] @haftamudesta Great work adding the index and show method, however by defining them at this milestone causes an error when trying to access the show page as shown below, This also hindered me from checking the posts index and show pages
`hint You can leave all the methods empty for now'
![image](https://github.com/haftamudesta/Blog-App/assets/61408860/2c439678-e825-40ae-8117-87d2fb919450)